Gems/ROS2/Code/Include/ROS2/Lidar/RaycastResults.h

```diff
@@ -52,7 +57,7 @@ namespace ROS2
 
     struct SegmentationIds
     {
-        int32_t m_entityId;
+        AZ::s32 m_entityId;
```

Comment:
What is the reason for this change?

Reply:
This is for consistency and expands to int32_t.